tests/fps_tests.py

```diff
@@ -296,7 +324,6 @@ def test_servicesets_overlap(self):
                     service_sites=["https://service1.com"]),
                     
```

Comment:
Looks like `expected_sets` is no longer used here, and probably in other test cases as well.

Reply:
It's now been removed wherever extraneous. Thank you.